VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -684,6 +687,13 @@ def get_shell_script_log(self):
             self.log("Can't receive shell log file: " + str(e))
             return lines
 
+    def update_settings_file(self):
+        self.log("removing the protected settings" + json.dumps(self._context._config))
```

Comment:
the config value log is intentionally kept. i will remove it before final checkin, after all PR comments are resolved